script/deploy

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env ruby
```

Comment:
Yeah. Python is more ubiquitous, too. I'll think on it. For now, we can probably go ahead and just merge this as-is so that I can verify that `HEAD` builds properly. 😜